### ISSUE TITLE
Export / Print Control supported ? 

### ISSUE THREAD

**Issue by Martenz** [eodaGmbH/py-maplibregl#84]
Is this plugin already supported?

https://github.com/watergis/maplibre-gl-export

or can it be in future ?

Thanks

Martino

**Comment by crazycapivara** [comment]
Nope, it is not supported yet but I thin I can add a download option in the future.
